src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -136,30 +139,31 @@ extern "C" {
 	}
 
 	NODELOOPPLUGIN_DECL int ibmras_monitoring_plugin_init(const char* properties) {
-		return 0;
+       uv_check_init(uv_default_loop(), &check_handle);
```

Comment:
If I had to guess, I'd say you called `uv_check_init()` again before the `uv_close()` call completed, corrupting libuv's internal structures.  Init once and start/stop on demand is fine though; an inactive check handle consumes zero libuv resources.